e2e/app/tokenbridge.go

```diff
@@ -60,21 +55,29 @@ func setupTokenBridge(ctx context.Context, def Definition) error {
 
 	// Deploy the token
 
-	tokenAddr, receipt, err := omnitoken.Deploy(ctx, networkID, l1Backend)
+	tokenAddr, receipt, err := omnitoken.DeployIfNeeded(ctx, networkID, l1Backend)
 	if err != nil {
 		return errors.Wrap(err, "deploy omni token")
 	}
 
-	log.Info(ctx, "Deployed Omni Token", "chain", l1.Name, "addr", tokenAddr.Hex(), "block", receipt.BlockNumber)
+	if receipt != nil {
+		log.Info(ctx, "Deployed Omni Token", "chain", l1.Name, "addr", tokenAddr.Hex(), "block", receipt.BlockNumber)
+	} else if tokenAddr != networkID.Static().TokenAddress {
+		log.Warn(ctx, "Omni token already deployed, but not in network static", errors.New("missing static token addr"), "addr", tokenAddr.Hex())
```

Comment:
Wanna just fail the deployment in this case, since afaic tell it means we're doing something wrong?

Reply:
Hmm. It may not necessarily be wrong. Might just be a reminder to go put it in static, cause you forgot to last time. Useful in case where omega deploy fails, but some contracts were deployed.